From 1dddc5788c1709c300a57a00ec0709ac4011eb08 Mon Sep 17 00:00:00 2001
From: Yong Yao <yong.yao@intel.com>
Date: Tue, 31 Dec 2013 15:06:15 +0800
Subject: [ABT Mixins] co-exit for mixin and no-mixin solution

        AndroidBoard.mk, BoardConfig.mk, common.mk and init.common.rc
        were renamed with *_old.*

Change-Id: If5754b5a535290c1f360ed585f9e5affbc6361ae
Signed-off-by: Yong Yao <yong.yao@intel.com>
---
 AndroidBoard.mk |    2 +-
 BoardConfig.mk  |    2 +-
 cherrytrail.mk  |    2 +-
 3 files changed, 3 insertions(+), 3 deletions(-)

diff --git a/AndroidBoard.mk b/AndroidBoard.mk
index 6337d23..fa9eae4 100644
--- a/AndroidBoard.mk
+++ b/AndroidBoard.mk
@@ -1,7 +1,7 @@
 # make file for Cherrytrail
 #
 
-include device/intel/common/AndroidBoard.mk
+include device/intel/common/AndroidBoard_old.mk
 
 # Add socwatchdk driver
 -include linux/modules/debug_tools/socwatchdk/src/AndroidSOCWatchDK.mk
diff --git a/BoardConfig.mk b/BoardConfig.mk
index e27d5bb..42b2033 100644
--- a/BoardConfig.mk
+++ b/BoardConfig.mk
@@ -1,5 +1,5 @@
 
-include device/intel/common/BoardConfig.mk
+include device/intel/common/BoardConfig_old.mk
 
 FLASHFILE_NO_OTA := true
 
diff --git a/cherrytrail.mk b/cherrytrail.mk
index 494a831..cca8fd3 100644
--- a/cherrytrail.mk
+++ b/cherrytrail.mk
@@ -3,7 +3,7 @@ TARGET_BOARD_SOC := cherryview
 TARGET_BOOTLOADER_BOARD_NAME := $(TARGET_BOARD_PLATFORM)
 
 # Include common environnement
-include device/intel/common/common.mk
+include device/intel/common/common_old.mk
 
 # USB port turn around and initialization
 PRODUCT_COPY_FILES += \
-- 
1.7.4.1

